docs/user-guide/data-structures/creating-arrays-and-datasets.ipynb

```diff
@@ -0,0 +1,616 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipp as sc\n",
+    "v1d = sc.array(dims=['x'], values=[1, 2, 3, 4])\n",
+    "v2d = sc.array(dims=['x', 'y'], values=[[1, 2], [3, 4]])\n",
+    "v3d = sc.array(dims=['x', 'y', 'z'], values=[[[1, 2], [3, 4]],\n",
+    "                                             [[5, 6], [7, 8]]])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Alternatively, passing a NumPy array:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "a = np.array([[1, 2], [3, 4]])\n",
+    "v = sc.array(dims=['x', 'y'], values=a)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that *both* the NumPy array and Python lists are copied into the Scipp variable which leads to some additional time and memory costs.\n",
+    "See [Filling with a Value](#Filling-with-a-Value) for ways of creating variables without this overhead.\n",
+    "\n",
+    "The `dtype` of the variable is deduced automatically in the above cases.\n",
+    "The unit is set to `scipp.units.dimensionless` if the `dtype` is a numeric type (e.g. integer, floating point) or `None` otherwise (e.g. strings).\n",
+    "This applies to all creation functions, not just `scipp.array`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "v"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `dtype` can be overridden with the `dtype` argument (see [Data types](../../reference/dtype.rst)):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.array(dims=['x', 'y'], values=[[1, 2], [3, 4]], dtype='float64')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The unit can and almost always should be set manually (see [unit](../../reference/units.rst)):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.array(dims=['x', 'y'], values=[[1, 2], [3, 4]], unit='m')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Variances can be added using the `variances` keyword:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.array(dims=['x', 'y'], values=[[1.0, 2.0], [3.0, 4.0]],\n",
+    "         variances=[[0.1, 0.2], [0.3, 0.4]])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that `scipp.array` takes *variances* as an input while the output representation of the variable shows *standard deviations*.\n",
```

Comment:
I would recommend to change this to be more explicit. The output may not be the problem, but it is really easy to make a mistake and pass stddev's as input. Can we say something like:

```suggestion
    "Note that `scipp.array` takes *variances* as an input. If your input array of uncertainties stores *standard deviations* make sure to *square* them: `sc.array(..., variances=stddev_array**2)\n",
```